requirements/common.in

```diff
@@ -1,27 +1,23 @@
 # Packages that are shared between deployment and dev environments.
 gunicorn==20.0.4
-whitenoise[brotli]==5.0.1
-
-# Used by Whitenoise to provide Brotli-compressed versions of static files.
-Django==3.0.7
+whitenoise[brotli]  # Used by Whitenoise to provide Brotli-compressed versions of static files.
+Django==3.0.8
 celery==4.4.2   # 4.4.2 required to pass test_retryable_task_throws_retry, celery needed for data ingestion
-simplejson==3.17.0
+simplejson  # import simplejson
```

Comment:
@camd In general, we are removing the pinned versions so they will automatically upgrade when `pip-compile` is used.  This should have been done long before, but testing each one was too exhausting for a single PR